ROBOT_FRAME: "base_link"
MAX_VELOCITY: 0.4
MIN_VELOCITY: 0.2
MAX_GAMMA: 0.52
MIN_GAMMA: -0.52
MAX_ACCELERATION: 1.0
MAX_OMEGA: 1.0
CAL_L: 0.8633246
CAL_W: 1.036
